docs/operation/backup.md

```diff
@@ -0,0 +1,73 @@
+# Backup Data Using BanyanDB Backup Tool
+
+The backup command-line tool is used to backup BanyanDB snapshots to remote storage. The tool allows both one-time and scheduled backups.
+
+## Overview
+
+The backup tool performs the following operations:
+
+- Connects to a BanyanDB data node via gRPC to request the latest snapshots.
+- Determines the local snapshot directories based on the catalog type:
+  - **Stream Catalog:** Uses the `stream-root-path`.
+  - **Measure Catalog:** Uses the `measure-root-path`.
+  - **Property Catalog:** Uses the `property-root-path`.
+- Computes a time-based directory name (formatted as daily or hourly).
+- Uploads files that are not found in the remote storage to the specified destination.
+- Deletes orphaned files in the remote destination that no longer exist locally.
```

Comment:
As the directory name includes time, what causes the orphaned files to exist? The backup happened twice in one hour(or other time style)?

Reply:
The immutable files are deleted locally. To my knowledge, only the old and sufficiently large files can remain until the next backup time, which is one hour later. Most small files will be cleared up in the remote.

Reply:
I saw your doc about time-style, and backup period, which could be per hour. So, to delete the old files, mostly because the time-style is in day, and we run backup multiple times per day, right?

Reply:
Absolutely. By setting the `schedule` to `hourly` and the `time-style` to `daily`, the remote destination will be uploaded 24 times a day. This is my preferred strategy, and I plan to use it as the default setting for the Banyandb Helm chart.